packages/world/src/utils.sol

```diff
@@ -5,7 +5,7 @@ import { StoreSwitch } from "@latticexyz/store/src/StoreSwitch.sol";
 import { ResourceSelector } from "./ResourceSelector.sol";
 import { SystemRegistry } from "./Tables.sol";
 
-library Utils {
+library SystemUtils {
```

Comment:
renamed this to make the purpose a bit clearer, and to add more utility functions to the same file. (Unfortunately it's not possible to turn the `SystemUtils.systemNamespace()` function into a free function because it uses `address(this)`)

Reply:
Undid this change - felt unclean to have a mixed utils file, so i moved `revertWithBytes` to its own file. Still unhappy with this file being called just "Utils", but changing that should be its own PR.